src/dicarlo_lab_to_nwb/conversion/probe.py

```diff
@@ -180,7 +180,7 @@ def build_probe_group(
 
         # We leave a margin of 400 um between probes and the border
         margin_um = 400
-        probe.create_auto_shape(probe_type="circular", margin=margin_um)
+        probe.create_auto_shape(probe_type="rect", margin=margin_um)
```

Comment:
For this you need to re-update probeinterface as we added this very recently:

https://github.com/SpikeInterface/probeinterface/pull/279

But I only wanted to showcase it to you so I think this is probably safer, let's keep rect for a while.